packages/ts-invariant/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ts-invariant",
-  "version": "0.6.1",
+  "version": "0.6.2",
```

```suggestion
  "version": "0.6.1",
```
Lerna takes care of version bumps in this repo.

---

Oops - thanks @benjamn - forgot about Lerna! 😅